apigee_sync.go

```diff
@@ -235,6 +249,14 @@ func downloadDataSnapshot() {
 	scopes = append(scopes, apidInfo.ClusterID)
 	resp := downloadSnapshot(scopes)
 
+	knownTables = extractTablesFromSnapshot(resp)
+
+	db, err := data.DBVersion(resp.SnapshotInfo)
+	if err != nil {
+		log.Panicf("Database inaccessible: %v", err)
```

Comment:
good question, I'll look into it.  wouldn't thread safety be the responsibility of the data service?